packages/cfd/src/Containers/__tests__/cfd-password-manager-modal.spec.js

```diff
@@ -0,0 +1,354 @@
+
+    it('should render the DMT5 password modal', () => {
+        renderwithRouter(<CFDPasswordManagerModal {...mock_props} />);
+        expect(screen.getByText(/Manage DMT5 password/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/DMT5 password/i)[0]).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5OnePassword/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/DMT5 password/i)[1]).toBeInTheDocument();
+        expect(screen.getByText(/Investor password/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/DMT5 password/i)[2]).toBeInTheDocument();
+        expect(
+            screen.getByText(/Use this password to log in to your DMT5 accounts on the desktop, web, and mobile apps/i)
+        ).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Change DMT5 password/i })).toBeInTheDocument();
+    });
+
+    it('should render change-password-confirmation-modal if Change DMT5 password button is clicked', () => {
+        renderwithRouter(<CFDPasswordManagerModal {...mock_props} />);
+        fireEvent.click(screen.getByRole('button', { name: /Change DMT5 password/i }));
+        expect(screen.getByText(/Confirm to change your DMT5 password/i)).toBeInTheDocument();
+        expect(screen.getByText(/This will change the password to all of your DMT5 accounts/i)).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Cancel/i })).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Confirm/i })).toBeInTheDocument();
+    });
+
+    it('should render SentEmailModal if confirm button is clicked', async () => {
+        renderwithRouter(<CFDPasswordManagerModal {...mock_props} />);
+        fireEvent.click(screen.getByRole('button', { name: /Change DMT5 password/i }));
+        fireEvent.click(screen.getByRole('button', { name: /Confirm/i }));
+        expect(await screen.findByText(/We've sent you an email/i)).toBeInTheDocument();
+
+        expect(
+            await screen.findByText(/Please click on the link in the email to change your DMT5 password./i)
+        ).toBeInTheDocument();
+
+        expect(await screen.findByText(/Didn't receive the email?/i)).toBeInTheDocument();
+    });
+
+    it('should render the password modal for Deriv X', () => {
+        renderwithRouter(<CFDPasswordManagerModal {...mock_props} platform='dxtrade' />);
+        expect(screen.getByText(/Manage Deriv X password/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcDxtradeOnePassword/i)).toBeInTheDocument();
+        expect(screen.getAllByText(/Deriv X password/i)[0]).toBeInTheDocument();
+        expect(
+            screen.getByText(/Use this password to log in to your Deriv X accounts on the web and mobile apps/i)
+        ).toBeInTheDocument();
+        expect(screen.getByRole('button', { name: /Change Deriv X password/i })).toBeInTheDocument();
+    });
+
+    it('should render change password confirmation if change Deriv X button is clicked', () => {
```

Comment:
```suggestion
    it('should render change password confirmation modal if change Deriv X button is clicked', () => {
```
`change-password-confirmation-modal` above can be also renamed to `change password confirmation modal` for conformity 😊 (*the name of the component `ChangePasswordConfirmation` would also do)